connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectStandalone.java

```diff
@@ -60,11 +60,15 @@ public static void main(String[] args) throws Exception {
             Exit.exit(1);
         }
 
+        Time time = Time.SYSTEM;
+        log.info("Kafka Connect standalone worker initializing ...");
+        long initStart = time.hiResClockMs();
```

Comment:
Why does `ConnectDistributed` log worker info but standalone doesn't?

Reply:
ha! good catch! Lost in translation. I would have sworn I saw the info when I ran both modes. Fixed.